src/components/base/dialog.tsx

```diff
@@ -31,24 +31,27 @@ DialogOverlay.displayName = DialogPrimitive.Overlay.displayName
 
 const DialogContent = React.forwardRef<
   React.ElementRef<typeof DialogPrimitive.Content>,
-  React.ComponentPropsWithoutRef<typeof DialogPrimitive.Content>
->(({ className, children, ...props }, ref) => (
+    { showCloseButton?: boolean } &  React.ComponentPropsWithoutRef<typeof DialogPrimitive.Content>
+>(({ className, children,showCloseButton = true, ...props }, ref) => (
```

Comment:
Make it false by default so that if we have a dialog with the title, we show only 1 button by default